assets/setup-wizard/style.scss

```diff
@@ -255,7 +255,9 @@ $page-background: #b7c9be;
 			max-width: 414px;
 
 			&--large {
-				max-width: 538px;
+				max-width: 558px;
+				padding-right: 20px;
```

Comment:
The padding was added because the next content (site screenshot) comes over the text when scrolling, and the "effect" around the image has transparency, so depending on the resolution the text could appear in that transparent part while scrolling.